src/autolinks/autolinks.utils.ts

```diff
@@ -259,12 +313,28 @@ export function getBranchAutolinks(branchName: string, refsets: Readonly<RefSet[
 				if (existingIndex != null) {
 					index = Math.min(index, existingIndex);
 				}
+				console.log(
+					JSON.stringify(match.value),
+					match.value.groups.numberChunk,
+					match.value.groups.numberChunkBeginning,
+					match.value.input,
+					match.value.groups.issueKeyNumber,
+				);
```

Comment:
This console log should be removed. Likely left over from when you were testing it locally.

Reply:
removed